handlers/setup.js

```diff
@@ -0,0 +1,128 @@
+const axios = require("axios")
+const cliColor = require("cli-color")
+const { Client, GatewayIntentBits } = require("discord.js")
+const fs = require("fs")
+const readline = require('readline').createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+const questions = [
+    "Please enter your panel name: ",
+    "Please enter your panel URL: ",
+    "Please enter your panel API key: ",
+    "Please enter your bot token: ",
+    "Please enter your channel ID: "
+];
+
+const answers = [];
+
+const isValidURL = (url) => {
+    try {
+        new URL(url);
+        return true;
+    } catch (error) {
+        return false;
+    }
+};
+
+module.exports = function Setup() {
+    console.log(cliColor.cyanBright("Welcome to PteroStats!"))
+    console.log(cliColor.yellow("Please fill in the following credentials to set up the app.\n "));
+
+    const askQuestion = (index) => {
+        if (index < questions.length) {
+            console.log(questions[index]);
+
+            readline.question('> ', answer => {
+                let isValid = true;
+
+                if (index === 1 && !isValidURL(answer)) {
+                    console.log(cliColor.redBright('❌ Invalid Panel URL. Please enter a valid URL. Example Correct URL: "https://panel.example.com"'));
+                    isValid = false;
+                } else if (index === 2 && !/^(plcn_|ptlc_|peli_|ptla_)/.test(answer)) {
+                    console.log(cliColor.redBright("❌ Invalid Panel API key. It must start with 'plcn_' or 'ptlc_'."));
+                    isValid = false;
+                } else if (index === 4 && !/^\d+$/.test(answer)) {
+                    console.log(cliColor.redBright("❌ Invalid Channel ID. It must be a number."));
+                    isValid = false;
+                }
+
+                if (index === 2 && /^(peli_|ptla_)/.test(answer)) console.log(cliColor.yellow("The use of Application API keys are deprecated, you should use Client API keys"));
+
+                if (isValid) {
+                    answers.push(isValidURL(answer) ? new URL(answer).origin : answer);
+                    askQuestion(index + 1);
+                } else {
+                    askQuestion(index);
+                }
+            });
+        } else {
+            axios(`${new URL(answers[1]).origin}/api/application/nodes?include=servers,location,allocations`, {
+                method: "GET",
+                headers: {
+                    "Content-Type": "application/json",
+                    "Authorization": `Bearer ${answers[2]}`
+                },
+            }).then(() => {
+                console.log(" \n" + cliColor.green("✓ Valid Panel Credentials."));
+                const client = new Client({
+                    intents: [GatewayIntentBits.Guilds]
+                })
+
+                client.login(answers[3]).then(async () => {
```

Comment:
The enum suggested above can be done here too
```suggestion
                client.login(answers[Question.whatever]).then(async () => {
```